### DISCUSSION TITLE
macOS 10.13.6 hostname doesn't match computer name

### DISCUSSION THREAD

**Discussion by ghost** [amidaware/tacticalrmm#1747] [Q&A] [answered]
I have two machines, both macOS 10.13.6, that appear named in TRMM as `localhost` but in mesh and other software the proper name is displayed (ie `joe-bloggs`). 

How / where to fix this?

**Comment by dinger1986** [comment]
do you have code signing/an active sponsorship?

**Reply by ghost** [reply]
Not yet. I presume that's the answer to the problem?

**Reply by dinger1986** [reply, marked as answer]
the only supported route for mac and linux is with code signing/active sponsorship
